lib/omise/OmiseSearch.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+require_once dirname(__FILE__).'/res/OmiseApiResource.php';
+
+/**
+ * This class is not intended to be used directly from client code.
+ *
+ * @see OmiseCharge::search()
+ */
+class OmiseSearch extends OmiseApiResource
+{
+    const ENDPOINT = 'search';
+
+    private $dirty = true;
+    private $scope = null;
+    private $query = '';
+    private $conditions = array();
+    private $page = 1;
+    private $order = 'chronological';
```

Comment:
@iwat I think library should just reflect to the API as much as possible unless user config it.
For these params, how about set to null and lets API handle the result by itself?